README.md

```diff
@@ -0,0 +1,18 @@
+# handbook
+Handbook for interacting with the DANDI archive.
+
+## DANDI Style Sheet
```

Comment:
That's a fair point. I used style "sheet" instead of "guide" because (for example) we have a comprehensive style guide at Kitware, and I used "sheet" to distinguish that from this more project-specific set of guidelines. I will rename it to "DANDI Style Guidelines".